src/AppInstallerRepositoryCore/Rest/RestClient.cpp

```diff
@@ -20,6 +20,8 @@ namespace AppInstaller::Repository::Rest
     // Supported versions
     std::set<Version> WingetSupportedContracts = { Version_1_0_0 };
 
+    constexpr std::string_view WindowsPackageManagerHeader = "Windows-Package-Manager"sv;

```

Comment:
>constexpr std::string_view WindowsPackageManagerHeader = "Windows-Package-Manager"sv; [](http://example.com/codeflow?start=4&length=85)

nit: probably this would move to CommonRestConstants.h during integration with my rest 1.1 change